sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -93,13 +93,12 @@ public DateTime AsDate()
                 throw new InvalidOperationException($"Cannot get field as Date.  Field value's type is {Type}.");
             }
 
-            DateTime date = default;
-            if (!DateTime.TryParse(_fieldValue.ValueDate, out date))
+            if (!_fieldValue.ValueDate.HasValue)
             {
                 throw new InvalidOperationException($"Cannot parse Date value {_fieldValue.ValueDate}.");
             }
 
-            return date;
+            return _fieldValue.ValueDate.Value.UtcDateTime;
```

Comment:
@annelo-msft based on https://github.com/Azure/azure-sdk-for-net/issues/11120#issuecomment-612175895 
is it ok to use `UtcDateTime` here?

Reply:
The return type is correct.  I'm not 100% certain what time zone is returned for receipts.  Have you validated that the time is the same as the time returned for the receipt?

Reply:
Yes. so far no different